web/docs/heroku.md

```diff
@@ -18,14 +18,41 @@ git commit -m "Initial version"
 ## Build and deploy from Git to a Docker container
 
 1. Login to Heroku Container Registry: `heroku container:login`
-1. Create an app in Heroku using `heroku create -a my-app-name -s container`. This will configure Heroku with a container-based app and create a git remote named `heroku` for deploying the app.  It will also return the URL to where the app will run when deployed, in the form of `https://my-app-name.herokuapp.com`
+1. Create an app in Heroku using `heroku create -a my-app-name -s container`. This will configure Heroku with a container-based app and create a git remote named `heroku` for deploying the app.  It will also return the URL to where the app will run when deployed, in the form of:
+
+    ```text
+    https://my-app-name.herokuapp.com
+    ```
+
+1. To create a new app in the Partner Dashboard or to link the app to an existing app, run the following command using your preferred package manager:
+
+    Using yarn:
+
+    ```shell
+    yarn run info --web-env
```

Comment:
Did we add this to the root package.json?

Reply:
Yeah, there's an `info` in the root `package.json` of both Node and PHP repos, looks like it was added almost 3 weeks ago!